Makefile.in

```diff
@@ -110,6 +110,15 @@ doc: warnroot
 util:
 	cd $@; $(MAKE) $(MFLAGS)
 
+clean-util:
+	cd $@; $(MAKE) $(MFLAGS) clean
```

Comment:
cd $@ does 'cd clean-util' but no such dir exists.
same with the two following targets

Reply:
Fixed. I must have made those changes directly to the Makefile rather than the Makefile.in and missed them. Fixed in latest commit.